apps/docs/assets/scripts/icons-gallery.js

```diff
@@ -1,48 +1,72 @@
 const BASE_URL = 'https://icon.resources.vonage.com'; // namespaced as 3f7739a0-a898-4f69-a82b-ad9d743170b6 on icons.resources.vonage.com
 const ICON_SET_VERSION = '4.2.1';
 
-const ICONS_TO_SHOW = 21;
+const NUM_TO_SHOW = 21;
 
 let jsonData;
 let index = 0;
-let iconsShown = ICONS_TO_SHOW;
+let numShown = NUM_TO_SHOW;
 
 fetchJSONData();
 
 async function fetchJSONData() {
   try {
     const response = await fetch(`${BASE_URL}/v${ICON_SET_VERSION}/manifest.json`);
     jsonData = await response.json();
+    addCategoryOptions(jsonData);
     showIcons(jsonData);
   } catch (err) {
     console.error(err);
   }
 }
 
+function addCategoryOptions(data) {
+  const categoryNamesArray = [];
+  for (let i = 0; i < data.length; i++) {
+    const categoryName = data[i].tag.filter(s => s.includes('category_'))[0]?.substring(9);
+    categoryNamesArray.push(categoryName);
+  }
+  uniqueCategories(categoryNamesArray);
+}
+
+function uniqueCategories(categoryNamesArray) {
```

Comment:
🤡 Nitpick: it could be a bit more clear what it does

```suggestion
function addUniqueCategories(categoryNamesArray) {
```